components/tabs/tabs.js

```diff
@@ -378,7 +378,12 @@ class Tabs extends LocalizeStaticMixin(ArrowKeysMixin(RtlMixin(LitElement))) {
 			const handleTransitionEnd = (e) => {
 				if (e.propertyName !== 'max-width') return;
 				tab.removeEventListener('transitionend', handleTransitionEnd);
-				this._tabInfos.splice(this._tabInfos.findIndex(info => info.id === tabInfo.id), 1);
```

Comment:
I'm a little confused. We're already [including a findIndex](https://github.com/Brightspace/brightspace-integration/blob/master/js/polyfills.js#L2) polyfill in BSI...

Reply:
Oh, I was testing in the core demo pages where it's not included.  Hmm.  If we want to just rely on that, it would simplify this a bit.